src/components/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const PLAYER_MINIMUM_GAMES_REQUIRED = 10;
+export const PLAYER_MINIMUM_WINS_REQUIRED = 5;

```

Comment:
nit: can we put in a comment where this is used? Not clear why we would need to compare to min wins